website/docs/externalwarnings.md

```diff
@@ -0,0 +1,34 @@
+Controls the number of warnings that are shown by the compiler. This setting only applies to visual studio 17 and newer. It will control the warnings shown for includes given by sysincludedirs
```

Comment:
```suggestion
Controls the number of warnings that are shown by the compiler. This setting only applies to Visual Studio 17 and newer. It will control the warnings shown for includes given by [sysincludedirs](sysincludedirs.md).
```